resources/measures/ResidentialAirflow/measure.xml

```diff
@@ -2,8 +2,8 @@
   <schema_version>3.0</schema_version>
   <name>residential_airflow</name>
   <uid>7d2adf5c-9995-49be-b662-a23d2452bb36</uid>
-  <version_id>a6d7e12f-5ff2-4415-954a-8bc75d4a9e4b</version_id>
-  <version_modified>20201105T190401Z</version_modified>
+  <version_id>a2adcec5-f390-48a7-ad98-9dd0b50fc58b</version_id>
+  <version_modified>20201113T224354Z</version_modified>
```

Comment:
The changes to this file look accidental. Probably not a big deal, but make sure to only commit changes related to the PR.

Reply:
This might be ok. Could be result of running `update_measures`.